docs/resources/subgraph.md

```diff
@@ -36,16 +36,16 @@ resource "cosmo_subgraph" "test" {
 ### Required
 
 - `name` (String) The name of the subgraph.
-- `namespace` (String) The namespace in which the subgraph is located.
 - `routing_url` (String) The routing URL of the subgraph.
 
 ### Optional
 
-- `headers` (List of String) Headers for the subgraph.
 - `is_event_driven_graph` (Boolean) Indicates if the subgraph is event-driven.
 - `is_feature_subgraph` (Boolean) Indicates if the subgraph is a feature subgraph.
 - `labels` (Map of String) Labels for the subgraph.
+- `namespace` (String) The namespace in which the subgraph is located.
```

Comment:
Is this auto generated?

Reply:
Yes it comes from the schemas: https://github.com/wundergraph/terraform-provider-cosmo/blob/fabbe314d0c1e5679d0fa0ba080684683ae8ee53/internal/service/federated-graph/resource_cosmo_federated_graph.go#L58